runme.sh

```diff
@@ -262,7 +263,7 @@ mkimage -f kernel2160cex7.its kernel-lx2160acex7.itb
 ###############################################################################
 echo "Assembling image"
 cd $ROOTDIR/
-IMG=lx2160acex7_${SPEED}_${SERDES}_${BOOT}.img
+IMG=lx2160acex7_${BOOT_LOADER}_${SPEED}_${SERDES}_${BOOT}.img
```

Comment:
This needs a `README.md` update for instructions on how to use the scripts/output.